Networking/NetworkingTests/Mapper/ProductMapperTests.swift

```diff
@@ -189,6 +190,23 @@ final class ProductMapperTests: XCTestCase {
         XCTAssert(productImage.alt?.isEmpty == true)
     }
 
+    /// Test that product downloadable files are properly mapped.
+    ///
+    func test_that_product_downloadable_files_are_properly_mapped() {
+        let product = mapLoadProductResponse()
+
+        guard let files = product?.downloads else {
+            XCTFail("Failed to parse product category")
+            return
+        }
+        XCTAssertEqual(files.count, 2)
+
+        let productDownloadableFile = files[0]
```

Comment:
@pmusolino , thanks a lot for the review and pointers regarding CD model versioning. 😃 

Regarding this one, I am not really sure about the suggestion. But what I have done here is,

- Unwrapped the `product?.downloads` into `files`. So `files` is already unwrapped.
- `XCTAssertEqual(files.count, 2)` this line confirm is we have 2 files or not. 
- And `product?.downloads` is also non-optional

So it won't allow to unwrap a non-optional again like this `guard let productDownloadableFile = files?[0]` or this `if let productDownloadableFile = files?[0]`

But I can check the count manually. like so,
`
if files.count < 1 {
     XCTFail("Missing product downloadable file")
     return
}
`
Although it was a nice catch. A possible crash point. 👏

Reply:
I wasn't noticed that you was unwrapping it, so it's totally fine to remove this check 
```
if files.count < 1 {
            XCTFail("Missing product downloadable file")
            return
        }
```
since you are also checking `XCTAssertEqual(files.count, 3)`. 👍

Reply:
OK.